01_temperature_processing.R

```diff
@@ -75,7 +80,8 @@ logchar<-unique(audits[,c('LOGGER_ID','PARAMETER')])
 names(logchar)<-c('log','char')
 
 # Load the QC criteria for continuous data
-ConQC <- read.csv('//deqlab1/wqm/Volunteer Monitoring/datamanagement/R/ContinuousDataReview/ConQC.csv')
+load('//deqlab1/wqm/Volunteer Monitoring/datamanagement/R/ContinuousDataReview/ConQC.RData')
```

Comment:
This file is not in this location currently. Perhaps will all the back and forth with commits it was missed? Since we are working within a project we don't need to use the full path name either, for future reference.